graphdatascience/tests/integration/test_graph_ops.py

```diff
@@ -652,7 +652,7 @@ def test_graph_streamRelationshipProperties_with_arrow(gds: GraphDataScience) ->
 
 
 @pytest.mark.compatible_with(min_inclusive=ServerVersion(2, 2, 0))
-def test_graph_relationshipProperties_stream_with_arrow(gds: GraphDataScience) -> None:
+def test_graph_relationshipProperties_stream(gds: GraphDataScience) -> None:
```

Comment:
I think we want to keep 'with_arrow` in the name. It uses `gds`, that has arrow enabled. `gds_without_arrow` is the one without.